platforms/core-configuration/configuration-cache/src/crossVersionTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheProblemsTapi813PlusCrossVersionTest.groovy

```diff
@@ -85,7 +85,9 @@ class ConfigurationCacheProblemsTapi813PlusCrossVersionTest extends ToolingApiSp
             definition.id.group.name == "configuration-cache"
             definition.severity == Severity.ERROR
             (originLocations[0] as LineInFileLocation).path == "build file 'build.gradle'" // FIXME: the path should not contain a prefix nor extra quotes
-            ((targetVersion.baseVersion >= GradleVersion.version("8.14") ? contextualLocations[0] : originLocations[1]) as LineInFileLocation).path == buildFileLocation(buildFile, targetVersion)
+            if (targetVersion.baseVersion < GradleVersion.version("8.14")) {
+                (originLocations[1] as LineInFileLocation).path == buildFileLocation(buildFile, targetVersion)
```

Comment:
🤔  Is the information becoming worse now, since we have the "FIXME" path?

Reply:
no the FIXME is old actually, I'd address that separately.
That is related to configuration cache problems, they are creating the location information in this limited way.